src/Notifications/HubHelpers.cs

```diff
@@ -14,10 +14,12 @@ public static class HubHelpers
         string notificationJson,
         IHubContext<NotificationsHub> hubContext,
         IHubContext<AnonymousNotificationsHub> anonymousHubContext,
+        ILogger logger,
         CancellationToken cancellationToken = default(CancellationToken)
     )
     {
         var notification = JsonSerializer.Deserialize<PushNotificationData<object>>(notificationJson, _deserializerOptions);
+        logger.LogInformation("Sending notification: {NotificationType}", notification.Type);
```

Comment:
🎨 If you're adding a log, consider adding additional data points if it's valuable.

Reply:
did you have any suggestions? at the moment what we're looking for is just number of messages.

In principle this gives us update types, too, but I don't want to go so far as to log the target of updates, since userIDs are pii